tests/e2e/scale/test_ceph_pod_respin_in_scaled_cluster.py

```diff
@@ -25,9 +25,19 @@ def fioscale(request):
         node_selector=constants.SCALE_NODE_SELECTOR
     )
     fioscale.create_scale_pods(
-        scale_count=1500, pods_per_iter=10, io_runtime=36000,
+        scale_count=350, pods_per_iter=10, io_runtime=36000,
```

Comment:
@ramkiperiy We used to run IO on 1500 pods for 10 hours earlier and now we run,
IOs on 350 pods for 10 hours, followed by the creation of 350 pods without IOs and then 400 pods with ~1 hour and 400 pods without IO. 

We'll simply use 400 pods for each iteration and run IOs for 18000 seconds alternatively. IMO, It is okay to cross 1500 pods. (we anyway don't run IOs on all of them)

Also, is there any reason we run IO for such a long time?